crates/interpreter/src/interpreter/memory.rs

```diff
@@ -183,17 +179,9 @@ impl Memory {
     }
 }
 
-/// Rounds up `x` to the closest multiple of 32. If `x % 32 == 0` then `x` is returned.
-#[inline]
-pub(crate) fn next_multiple_of_32(x: usize) -> Option<usize> {
```

Comment:
Moved to `revm_primitives`

Reply:
Related only to Interpreter so it was left there, does not make sense imo to move it to primitives